tests/bgp/test_4-byte_asn_community.py

```diff
@@ -0,0 +1,231 @@
+'''
+
+This script is to Verify applied communities manipulate traffic as
+expected between 4-byte and 2-byte AS neighbors.
+
+Step 1: Configure DUT and neighbor with 4Byte ASN
+Step 2: Verify 4-byte BGP session between DUT and neighbor is established
+
+'''
+import logging
+
+import pytest
+import time
+import textfsm
+from tests.common.config_reload import config_reload
+
+logger = logging.getLogger(__name__)
+dut_4byte_asn = 400003
+neighbor_4byte_asn = 400001
+bgp_sleep = 60
+bgp_id_textfsm = "./bgp/templates/bgp_id.template"
+
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+
+@pytest.fixture(scope='module')
+def setup(tbinfo, nbrhosts, duthosts, enum_frontend_dut_hostname, enum_rand_one_frontend_asic_index, request):
+    # verify neighbors are type sonic
+    if request.config.getoption("neighbor_type") != "sonic":
+        pytest.skip("Neighbor type must be sonic")
+    duthost = duthosts[enum_frontend_dut_hostname]
+    asic_index = enum_rand_one_frontend_asic_index
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+    dut_asn = tbinfo['topo']['properties']['configuration_properties']['common']['dut_asn']
+    neigh = duthost.shell("show lldp table")['stdout'].split("\n")[3].split()[1]
```

Comment:
this `neigh` is a format-based variable, if output of `show lldp table` changed, we no longer can get the correct neighbor name, can you add logger.debug to print this `neigh`?

Reply:
I added a logging line to print this variable after it is set.